### PR TITLE
🚸 Warehousing is no longer default behavior

### LINKED ISSUE #612: Should warehousing default to off?
I want to start a discussion about this: I think warehousing should default to off. 

Unless I have a special case where I *need* to warehouse the output of my ingest run, i always run `kidsfirst test` and `kidsfirst ingest` with the `--no_warehouse` flag. 

I'm curious if any of the other ingest package developers are warehousing the outputs of their ingests and how they feel about defaulting warehousing to be off.

### PR DESCRIPTION
**Pull request by fiendish**
closes #612

### CODE REVIEW

File: kf_lib_data_ingest/app/cli.py [hunk 1]
```diff
@@ -197,10 +197,10 @@ def cli():
     "the target service. Overrides the resume_from setting.",
 )
 @click.option(
-    "--no_warehouse",
+    "--warehouse",
     default=False,
```
Comment (znatty22) [inline kf_lib_data_ingest/app/cli.py:201]: This means the `warehouse` kwarg is set to `False`  by default right? I can't remember click

**Review by gsantia — APPROVED** [review]
┌༼ຈل͜ຈ༽┘

File: kf_lib_data_ingest/app/cli.py (hunk 1 above)
Comment (fiendish) [reply kf_lib_data_ingest/app/cli.py:201]: It _should_ mean that :D

**Comment by Christina-J-Diaz** [comment]
✨ ✅

**Review by Christina-J-Diaz — APPROVED** [review]

File: kf_lib_data_ingest/app/cli.py (hunk 1 above)
Comment (fiendish) [reply kf_lib_data_ingest/app/cli.py:201]: I think a boolean toggle must always default to false, so it probably doesn't even need to be there. Like if it defaulted to true, how would you set it false right?

**Review by znatty22 — APPROVED** [review]

**Pull request merged**